### PR TITLE
Merge changes to make the purpose of file clearer

### LINKED ISSUE #102: New feature request
* **Is your feature request related to a problem? Please describe.**

Want to add more functionality. 

* **Describe the solution you'd like**

* **Describe alternatives you've considered**

* **Additional context**

### PR DESCRIPTION
**Pull request by RA-Manisha-Jundoosingh**
## Proposed changes

Merging additional text to make this file clearer to understand what it's about.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [ x] Documentation Update (change to naming or other documentation)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR._

- [ ] I have added unit tests that prove my fix is effective or that my feature works
- [ x] I have added necessary documentation (if appropriate)
- [ x] I have checked that my changes have not broken any other functionality
- [ ] I have linked to any issues this PR fixes 

## Points for review

Add checkboxes here for any specific aspects of your PR you would like to be checked in peer review. If you don't specify anything here, the reviewer will check:

- [ ] All of your unit tests pass
- [ ] Your code is clear and easy to understand
- [x ] You have documented any new features
- [ ] They can run the code you have written
- [ x] Your changes do not break any other functionality

## Who is reviewing this PR?

@ran-maciej-idzikowski

### CODE REVIEW

File: manisha-jundoosingh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+I'm on the GitHub training 30/06/23.
```
Comment (RA-Manisha-Jundoosingh) [inline manisha-jundoosingh:1]: Would you recommend this to Rail Analysis?

Comment (RA-Manisha-Jundoosingh) [reply manisha-jundoosingh:1]: @ran-maciej-idzikowski what do you think?

File: text_file.txt [hunk 2]
```diff
@@ -1 +1,12 @@
...
+<<<<<<< HEAD
+
+nice
+=======
+I'm on the GitHub training 30/06/23.
+
+I'm running the Github training.
+>>>>>>> 12444d42ebb496e237a46d962d6cc23069e0ef76
+
+yup
```
Comment (RA-Manisha-Jundoosingh) [inline text_file.txt:11]: @ran-maciej-idzikowski Maybe more useful if you comment you're on the training too!

**Review by RA-Manisha-Jundoosingh — COMMENTED** [review]
I've commented questions and suggested changes to the changes made on the text file.

File: manisha-jundoosingh (hunk 1 above)
Comment (ran-maciej-idzikowski) [reply manisha-jundoosingh:1]: 100%

If only for the code reviewing.

**Review by ran-maciej-idzikowski — APPROVED** [review]

**Pull request closed without merging**